doc/nrf/releases/release-notes-1.5.0.rst

```diff
@@ -0,0 +1,1132 @@
+  * Moved the :ref:`http_application_update_sample` sample from :file:`samples/nrf9160/http_application_update/` to :file:`samples/nrf9160/http_update/application_update`.
+  * :ref:`lib_download_client` library:
+
+    * Reintroduced the optional TCP timeout (enabled by default) on the TCP socket that is used for the download.
+      Upon timeout on a TCP socket, the HTTP download will fail and the ``ETIMEDOUT`` error will be returned via the callback handler.
+    * Added an option to set the hostname for TLS Server Name Indication (SNI) extension.
+      This option is valid only when TLS is enabled.
+
+  * :ref:`lib_date_time` library :
+
+    * Added an API to check if the Date-Time library has obtained a valid date-time. If the function returns false, it implies that the library has not yet obtained valid date-time to base its calculations and time conversions on, and hence other API calls that depend on the internal date-time will fail.
+
+
+  * :ref:`lib_fota_download` library - Enabled SNI TLS extension for HTTPS downloads.
+  * :ref:`lib_nrf_cloud` library:
+
+    * nRF Connect for Cloud FOTA replaced AWS Jobs as the FOTA mechanism for devices connected to nRF Connect for Cloud.
+    * Removed :option:`CONFIG_CLOUD_API` dependency from :option:`CONFIG_NRF_CLOUD_CONNECTION_POLL_THREAD`.
+    * Added a new API :c:func:`nrf_cloud_send` that can be used for sending pre-encoded data to specified endpoint topics in nRF Connect for Cloud.
+
+  * :ref:`at_cmd_parser_readme` library - The library can now parse AT command strings with negative numbers in the range supported by the int32_t type.
+  * :ref:`lib_azure_iot_hub` library - Improved the internal connection state handling.
+  * :ref:`lte_lc_readme` library - Added support for manufacturer-specific default eDRX/PSM values.
+  * :ref:`liblwm2m_carrier_readme` library - Updated to v0.10.2. See :ref:`liblwm2m_carrier_changelog` for details.
+
+DFU Target
+----------
+
+* Added:
+
+  * New target ``dfu_target_full_modem``, which is used for full modem firmware updates. This requires an external flash memory of a minimum of 2MB to work.
+
+* Updated:
+
+  * Renamed ``dfu_target_modem`` to ``dfu_target_modem_delta``.
+  * Moved the ``dfu_target`` code from :file:`subsys/dfu` to :file:`subsys/dfu/dfu_target`.
+  * Extracted the stream flash memory functionality from ``dfu_target_mcuboot`` to ``dfu_target_stream_flash`` to facilitate code reuse for other ``dfu_targets``, which write large objects to the flash memory.
+
+nRF5
+====
+
+The following changes are relevant for the nRF52 and nRF53 Series.
+
+nRF52832 SoC
+------------
+
+* Removed:
+
+  * ``esb`` subystem - Removed support for older SoC revision workarounds.
+
+    * Removed support for nRF52832 revision 1 workarounds for Errata 102, Errata 106, and Errata 107.
+    * Removed support for nRF52832 revision 2 workarounds for Errata 143.
```

Comment:
This line is incorrect, support for this workaround was re-introduced here: 2cd2c300cd81424fad582fbd72922549842ccc4d

Reply:
Removed